### ISSUE TITLE
WorldGuard Compatibility v14.13 error

### ISSUE THREAD

**Issue by ArtelGG** [SirBlobman/CombatLogX#109]
**Describe the bug**
I'm helping a friend set up a server for 1.8.8 _(Yes, I know... 1.8.8; I lost the argument)_ and I'm running into an error, the WorldGuard compatibility expansion is complaining during startup. By the looks of the error, it looks like there's a good possibility it could affect the functionality of the plugin. This issue is occurring on default configuration files for both WorldGuard and CLX.

**To Reproduce**
Start the server.

**Expected behaviour**
There should be no errors.

**Screenshots**
https://gist.github.com/ArtelGG/b1bd37ab014d5546c8f6583db9899dde

**Minecraft Version**
**CombatLogX Version:**
https://gist.github.com/ArtelGG/6fd9cdaef7b2c28ebdb8b8debd760eaf

**Additional context**
I didn't do any testing to check whether or not this actually affects the functionality.

**Comment by SirBlobman** [comment]
What WorldGuard version are you using?

**Comment by ArtelGG** [comment]
The latest one currently available for 1.8.8, 6.2.

**Comment by SirBlobman** [comment]
> The latest one currently available for 1.8.8, 6.2.

Please send a screenshot of `/version WorldGuard`

**Comment by ArtelGG** [comment]
Obtained from [here](https://dev.bukkit.org/projects/worldguard/files/956770).
```
version WorldGuard
[17:57:51 INFO]: WorldGuard version 6.1.3-SNAPSHOT;c904242
```

**Comment by SirBlobman** [comment]
WorldGuard flag registering method fixed in 10.0.0.0 builds. If this issue appears in those versions please reopen this issue :)
